container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -101,6 +107,10 @@ func CreateEngine(
 		consts.DockerSocketFilepath: consts.DockerSocketFilepath,
 	}
 
+	if serverArgs.OnBastionHost {
+		bindMounts[consts.HostEngineConfigDirToMount] = consts.EngineConfigLocalDir
```

Comment:
The directory on the bastion host containing the remote backend config.

Reply:
Same here, let's put that context in the code with a comment, explaining why we need this specific mount when we're on the bastion